docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -2625,3 +2625,18 @@ function sitenow_update_9067() {
     }
   }
 }
+
+/**
+ * Set new branding options theme setting.
+ */
+function sitenow_update_9068() {
+  $config_factory = \Drupal::configFactory();
+  $config = $config_factory->getEditable('uids_base.settings');
+  if ($config->set('style.style_selector', 'gray')) {
```

Comment:
Is this correct? If the config `style.style_selector` can be set to `gray` (returns TRUE), then set the option? Or should it be `$config->get('style.style_selector') === 'gray'`?

Reply:
Yes, you are correct - thanks!